cmd/controller-manager/main.go

```diff
@@ -165,13 +170,17 @@ func main() {
 
 	cListers := make(chan interface{}, 1)
 
-	if err := bindingController.Start(ctx, workers, cListers); err != nil {
-		setupLog.Error(err, "error starting the binding controller")
-		os.Exit(1)
+	if util.ShouldStartController(strings.ToLower(binding.ControllerName), ctlrsToStart) {
+		setupLog.Info("Starting controller", "name", binding.ControllerName)
```

Comment:
Since this line is for exactly one, known, controller, it could be simpler: `Info("Starting binding controller")`.